Source/Core/Core/AchievementManager.h

```diff
@@ -47,7 +54,18 @@ class AchievementManager
 
   rc_runtime_t m_runtime{};
   bool m_is_runtime_initialized = false;
+
   rc_api_login_response_t m_login_data{};
+
+  static constexpr int HASH_LENGTH = 33;
+  char game_hash[HASH_LENGTH];
```

Comment:
I don't see why this needs to be a class member, but if it is please `m_game_hash`. Also prefer `std::array` unless you have a very good reason not to.

Reply:
m_game_hash (dunno how I missed the m_ the first time, thanks for catching it) is populated via
`int rc_hash_generate_from_file(char hash[33], int console_id, const char* path)`
and is used in
```typedef struct rc_api_resolve_hash_request_t {
  /* Unused - hash lookup does not require credentials */
  const char* username;
  /* Unused - hash lookup does not require credentials */
  const char* api_token;
  /* The generated hash of the game to be identified */
  const char* game_hash;
}```
Don't really see much reason in converting it away from a char[] when I'm just going to be converting it back into a char[] when I use it.